tests/api_resources/PhoneNumberTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Telnyx;
+
+class PhoneNumberTest extends TestCase
+{
+    const TEST_RESOURCE_ID = '123';
+
+
+    public function testIsListable()
+    {
+        $this->expectsRequest(
+            'get',
+            '/v2/phone_numbers'
+        );
+        $resources = PhoneNumber::all();
+        $this->assertInstanceOf(\Telnyx\Collection::class, $resources);
+        $this->assertInstanceOf(\Telnyx\PhoneNumber::class, $resources[0]);
+    }
+
+    public function testIsUpdatable()
```

Comment:
Is there a delete method?

Reply:
This is how you delete a phone number

$x = PhoneNumber::retrieve("1199395468564497853");
$x->delete();